ci/windows-pkg-arrow-for-r.sh

```diff
@@ -54,6 +54,10 @@ wget https://dl.bintray.com/rtools/backports/mingw-w64-x86_64-snappy-1.1.7-2-any
 # double-conversion is only available in the Rtools4.0 builds, but apparently that's ok
 wget https://dl.bintray.com/rtools/mingw64/mingw-w64-x86_64-double-conversion-3.1.2-1-any.pkg.tar.xz
 wget https://dl.bintray.com/rtools/mingw32/mingw-w64-i686-double-conversion-3.1.2-1-any.pkg.tar.xz
+
+wget https://dl.bintray.com/rtools/mingw64/mingw-w64-x86_64-openssl-1.1.1.a-1-any.pkg.tar.xz
```

Comment:
This may not work, you may need a "backports" version of openssl. @jeroen might know.

Reply:
Also I'd be surprised that you could add a dependency and not have to edit the depends statement in `ci/PKGBUILD`: https://github.com/apache/arrow/blob/master/ci/PKGBUILD#L27

Reply:
Any idea @majetideepak?

Reply:
> Presumably there's a cmake flag for turning that off?
Not yet. Encryption is turn on as default when openssl is found. Any idea @majetideepak?

Reply:
Looks like we should add a flag to disable encryption support even if the system has OpenSSL.
@thamht4190 Can you rebase with master first?

Reply:
@majetideepak the rebase would take a lot of effort so I've merged latest code to my code.

Reply:
@nealrichardson there is a green build now. Do you still prefer to disable OpenSSL for Windows and R? The current state is that OpenSSL get included by default if CMake finds it in the environment (and Parquet build is ON).

Reply:
If the build is passing, I'm ok with leaving it on by default. Is there a way to turn it off if we do encounter unforeseen circumstances when submitting to CRAN?

Reply:
We don't and I also agree that we should have one. I will make this change.

Reply:
encryption wont be built by default unless `PARQUET_REQUIRE_ENCRYPTION` is specified.